ghcide/src/Development/IDE/Core/Rules.hs

```diff
@@ -35,7 +35,7 @@ import           Control.Concurrent.Extra
 import           Control.Exception
 import           Control.Monad.Extra
 import           Control.Monad.Reader
-import           Control.Monad.State
+import           Control.Monad.State.Strict
```

Comment:
This is the only change, TBH.

Reply:
If this is the only change please revert the other white space changes please

Reply:
Is there a reason for those aligned imports and unaligned imports?